src/renderer/store/modules/utils.js

```diff
@@ -292,51 +292,51 @@ const actions = {
     switch (strings[1].substring(0, 2)) {
       case 'se':
         if (singular) {
-          publicationString = publicationString.replace('%', payload.timeStrings.Second)
```

Comment:
The lower case has to be removed in the whole file, because of the same reason as before